engine/runtime/src/main/java/org/enso/interpreter/runtime/number/EnsoBigInteger.java

```diff
@@ -82,40 +82,50 @@ final boolean fitsInDouble() {
     return false;
   }
 
+  @ExportMessage
+  final boolean fitsInBigInteger() {
```

Comment:
Is there any reason for declaring all these methods as `final`, when the class itself is `final`?